agent/requirements/base.txt

```diff
@@ -252,22 +252,17 @@ multidict==5.2.0 \
     --hash=sha256:f9bef5cff994ca3026fcc90680e326d1a19df9841c5e3d224076407cc21471a1 \
     --hash=sha256:fc66d4016f6e50ed36fb39cd287a3878ffcebfa90008535c62e0e90a7ab713ae \
     --hash=sha256:fd77c8f3cba815aa69cb97ee2b2ef385c7c12ada9c734b0f3b32e26bb88bbf1d
-# via
-#   aiohttp
-#   yarl
-sentry-sdk==1.4.3 \
-    --hash=sha256:b9844751e40710e84a457c5bc29b21c383ccb2b63d76eeaad72f7f1c808c8828 \
-    --hash=sha256:c091cc7115ff25fe3a0e410dbecd7a996f81a3f6137d2272daef32d6c3cfa6dc
-# via -r requirements/base.in
-typing-extensions==3.10.0.2 \
-    --hash=sha256:49f75d16ff11f1cd258e1b988ccff82a3ca5570217d7ad8c5f48205dd99a677e \
-    --hash=sha256:d8226d10bc02a29bcc81df19a26e56a9647f8b0a6d4a83924139f4a8b01f17b7 \
-    --hash=sha256:f1d25edafde516b146ecd0613dabcc61409817af4766fbbcfb8d1ad4ec441a34
-# via async-timeout
+    # via
+    #   aiohttp
+    #   yarl
+sentry-sdk==1.5.1 \
```

Comment:
I supposed 1.4.3 -> 1.5.1 differences are not too bad:
https://github.com/getsentry/sentry-python/blob/master/CHANGELOG.md